modules/nf-core/fqtk/main.nf

```diff
@@ -0,0 +1,47 @@
+process FQTK {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::fqtk=0.2.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fqtk:0.2.0--h9f5acd7_0' :
+        'quay.io/biocontainers/fqtk:0.2.0--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(sample_sheet), path(fastq_dir)
+    val(fastq_readstructure_pairs)
+
+    output:
+    tuple val(meta), path('output/*R*.fq.gz')                       , emit: sample_fastq
+    tuple val(meta), path('output/demux-metrics.txt')               , emit: metrics
+    tuple val(meta), path('output/unmatched*.fq.gz')                , emit: most_frequent_unmatched
+    // TODO: Nathan is adding version printing with fqtk
+    path "versions.yml"                                             , emit: versions
+
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '0.2.0'
+    fastqs = fastq_readstructure_pairs.collect{it[2]/it[0]}.join(" ")
```

Comment:
What is `it[2]/it[0]` doing?

Reply:
This is a very necessary line! it[2] is the path to the directory containing the untar'd fastqs and it[0] is the name of the fastq. Example:
`[example_fastq.gz, 8B, path/to/working/dir/fastqs]` -> `path/to/working/dir/fastqs/example_fastq.gz`

Reply:
Ah, since `it[2]` is of type `Path`, then the `/` operator appends the filename.  Can you add a comment as such, it's super not clear from my first read.

Reply:
Yes, I will add a comment!

Reply:
Comment add in d30809f48f571b4e0bf2e0d20b9224d740957269